js-api-doc/importer.d.ts

```diff
@@ -308,6 +308,37 @@ export interface Importer<sync extends 'sync' | 'async' = 'sync' | 'async'> {
   nonCanonicalScheme?: string | string[];
 }
 
+/**
+ * An opaque branded type, representing the built-in Node Package Importer. This
+ * is used for type checking. Users should use the exposed `nodePackageImporter`
+ * and not instantiate their own.
+ *
+ * @category Importer
+ */
+type NodePackageImporter = {
+  _NodePackageImporterBrand: any;
```

Comment:
@nex3 This has a `no explicit any` eslint warning. Should I change this to `never` instead?

Reply:
I think `never` means that you're declaring that the object doesn't include the property. Does `unknown` work?

Reply:
`unknown` does work. I was suggesting `never` because the `_NodePackageImporterBrand` is a build-time only property, and never has to actually receive a value, and `never` is a stronger deterrent for users who may want to create their own. 

In  experimenting, `never` is more "the object requires the property is included, but it never has a value". Setting an object property to `never` doesn't seem to be documented by Typescript, as it is somewhat illogical, so I'm going to go with `unknown`.